### PR TITLE
Add Youtube class and search functionality

### PR DESCRIPTION
**Pull request by ImShady**
@themarkrizkallah Please review my PR! I added functionality for searching Youtube through a Youtube class and setup scaffolding for our main

### CODE REVIEW

File: youtube.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from oauth2client.tools import argparser
+import json
```
Comment (themarkrizkallah) [inline youtube.py]: I don't think this import is needed.

**Review by themarkrizkallah — CHANGES_REQUESTED** [review]
Changes to the syntax of print in youtube.query() should be made for consistency and "import json" isn't needed in main.py.

File: youtube.py (hunk 1 above)
Comment (ImShady) [reply youtube.py]: 😉

**Review by themarkrizkallah — APPROVED** [review]
Looks good. I will change the syntax of the print statements later.

**Pull request merged**